.github/workflows/tests.yml

```
@@ -31,52 +31,11 @@ on:
         default: "Manual trigger"
 
 jobs:
-  Tests:
-    runs-on: ubuntu-20.04
-    strategy:
-      fail-fast: false
-      matrix:
-        python-version: [3.9, 3.12]
-        db-service: [postgresql13]
-        search-service: [opensearch2]
-        include:
-          - db-service: postgresql13
-            DB_EXTRAS: "postgresql"
-
-          - search-service: opensearch2
-            SEARCH_EXTRAS: "opensearch2"
-
-        node-version: [16.x]
-    env:
-      DB: ${{ matrix.db-service }}
-      SEARCH: ${{ matrix.search-service }}
-      EXTRAS: tests,${{ matrix.SEARCH_EXTRAS }}
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v4
-        with:
-          node-version: ${{ matrix.node-version }}
-
-      - name: Run eslint test
-        run: ./run-js-linter.sh -i
-
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v5
-        with:
-          python-version: ${{ matrix.python-version }}
-          cache: pip
-          cache-dependency-path: setup.cfg
-
-      - name: Install dependencies
-        run: |
-          pip install ".[$EXTRAS]"
-          pip freeze
-          docker --version
-          docker-compose --version
-
-      - name: Run tests
-        run: ./run-tests.sh
+  Python:
+      uses: inveniosoftware/workflows/.github/workflows/tests-python.yml@master
+      with:
+        extras: "tests"
+  JS:
+    uses: inveniosoftware/workflows/.github/workflows/tests-js.yml@master
+    with:
+      translations-working-directory: ./invenio_app_rdm/assets/semantic-ui/translations/invenio_app_rdm
```

```suggestion
      translations-working-directory: ./invenio_app_rdm/theme/assets/semantic-ui/translations/invenio_app_rdm
```

There's are only `theme` assets apparently.